src/node_crypto.cc

```diff
@@ -568,84 +568,77 @@ void SecureContext::Init(const FunctionCallbackInfo<Value>& args) {
 
   if (args[0]->IsString()) {
     const node::Utf8Value sslmethod(env->isolate(), args[0]);
+    const std::string methodstr(*sslmethod);
```

Comment:
This does an extra allocation. Now, this is probably not a hot code path and there's a pretty good chance the small string optimization applies, but still (and it still copies memory, even with SSO.)

You could add a `bool Utf8Value::operator==(const char*) const` method instead.

Reply:
That's a great idea! PTAL.